pages/linux/xsetwacom.md

```diff
@@ -0,0 +1,23 @@
+# xsetwacom
+
+> Command line tool to change settings for Wacom pen tablets at runtime.
+
+- List all the available wacom devices. The device name is in the first column.
+
+`xsetwacom list`
+
+- Set Wacom area to specific screen. Get name of the screen with `xrandr`.
+
+`xsetwacom set "{{device name}}" MapToOutput {{screen}}`
+
+- Set mode to relative (like a mouse) or absolute (like a pen) mode.
+
+`xsetwacom set "{{device name}}" Mode "{{Relative|Absolute}}"`
+
+- Rotate the input (usefull for tablet-PC when rotating screen) by 0|90|180|270 degrees from "natural" rotation.
```

Comment:
(Copying comment originally left directly at 0cb4340, since that apparently doesn't appear in this PR)

> "useful" is spelled with a single L :)

> Also, I'd say "original orientation" or "default orientation" rather than "natural rotation".